requirements.txt

```diff
@@ -0,0 +1,10 @@
+# This file is redundant with pyproject.toml;
```

Comment:
Could this be auto-generated? e.g. by a pre-commit hook? It could be a source of problems if it gets out of sync with the .toml file.

Reply:
I actually generated this.. I guess we don't really need it for now, and yea I totally agree that this can be problematic as we add more dependencies.